nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentManager.java

```diff
@@ -322,4 +323,11 @@ void deleteExternalComment(CoreSession session, String entityId)
      */
     boolean hasFeature(Feature feature);
 
+    /**
+     * Gets the ancestor document ref for the given document.
+     * 
+     * @since 11.1
+     */
+    DocumentRef getAncestorRef(CoreSession session, DocumentModel documentModel);
```

Comment:
As you're making this public, it's worth explain what exactly is an ancestor in this context, because in general in Nuxeo an ancestor is the parent or the parent's parent or the parent's parent's parent etc. So there's no single ancestor, we usually talk about the set of ancestors for a document.

Or consider renaming the method to something else if it makes sense.